packages/modal/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@leafygreen-ui/modal",
+  "version": "0.9.0",
+  "description": "LeafyGreen UI Kit Modal",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build": "../../node_modules/.bin/webpack --config ../../webpack.config.js", 
+    "ts:emit": "tsc --project ./tsconfig.json"
+  },
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
```

Comment:
These should be peerDependencies.

Reply:
devDependencies*

Reply:
`Could not find a declaration file for module '@leafygreen-ui/icon'` when peerDeps

Reply:
Got it with the dev

Reply:
Yeah, sorry that was my bad.